app/templates/jinja2/previous_games.html

```diff
@@ -31,32 +31,32 @@
   </div>
 </header>
 
-<main class="ui-layout" id="landing">
-<br><br>
+<main class="full-height ui-layout" id="landing">
+  <aside class="ui-sidebar" id="side-nav">
+    <nav class="ui-menu">
+      <ul class="prev-games-list">
+      {% for prev_game in prev_games %}
+        <li><div class="ui-form-group pt-2">
+        <form class="mb-0" action="{{ url('previous_games') }}" method="post">
+            {{ csrf_input }}
+            <input class="previous-game font-weight-light btn" type="submit" value="{{prev_game.name}}" name="{{ prev_game.name }}">
+          </form>
 
-<aside class="container ui-sidebar" id="side-nav">
-  <nav class="ui-menu">
-    <ul class="ui-menu__section">
-    {% for prev_game in prev_games %}
-      <li><div class="ui-form-group pt-2">
-      <form class="mb-0" action="{{ url('previous_games') }}" method="post">
-          {{ csrf_input }}
-          <input class="btn btn-block btn-primary" type="submit" value="{{prev_game.name}}" name="{{ prev_game.name }}">
-        </form>
-        
-      </div></li>
-      {% endfor %}
-    </ul>
-  </nav>
-</aside>
+        </div></li>
+        {% endfor %}
+      </ul>
+    </nav>
+  </aside>
 
 
-<section class="container ui-layout__item" id="intro">
-<h2>{{ game_rendering }} Game Zombie Tree</h2>
-<div id="zombie-tree"></div>
-</section>
-<br><br>
-</main>
+  <section class="game-summary" id="intro">
+    <h1 class="prev-games-title">{{ game_rendering }} Game Zombie Tree</h1>
+    <div id="zombie-tree"></div>
+    <footer>
+      <p>&copy; UWaterloo Humans vs Zombies Society <time>2017</time> &ndash; <time><script>document.write(new Date().getFullYear())</script></time>.</p>
```

Comment:
Why is this hard coded?

Reply:
It's hardcoded in a bunch of places, same with the nav. As part of the next PR which is to redo the mobile website, I plan to move this all to reusable components.